kms.tf

```diff
@@ -6,7 +6,7 @@ locals {
 }
 
 data "aws_iam_policy_document" "kms_key_global" {
-  count = var.use_kms && var.kms_key_id == null ? 1 : 0
```

Comment:
If we're taking this out, we're removing the same (and the rest of the comments) in `variables.tf`, right?

Reply:
Correct